packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/meshes/skeletonPropertyGridComponent.tsx

```diff
@@ -82,7 +83,9 @@ export class SkeletonPropertyGridComponent extends React.Component<ISkeletonProp
             }
         } else {
             for (let index = 0; index < this._skeletonViewers.length; index++) {
-                this._skeletonViewers[index].mesh.reservedDataStore.skeletonViewer = null;
+                if (this._skeletonViewers[index].mesh) {
+                    (this._skeletonViewers[index].mesh as AbstractMesh).reservedDataStore.skeletonViewer = null;

```

Comment:
Why is the `as` needed here ? isnt the condition enough to be sure it is not null  ?

Reply:
IntelliJ is complaining if I remove it. Maybe I am missing something?

Reply:
@rafaeltavares helped me to fix it!